qml/pages/SettingsPage.qml

```diff
@@ -46,15 +45,29 @@ Page {
             TextSwitch {
                 checked: appSettings.sendByEnter
                 text: qsTr("Send message by enter")
-                description: qsTr("Send your message by pressing the enter key")
+                description: qsTr("Send your message by pressing the enter key.")
```

Comment:
If you do that, it invalidates all translations for this text. So, if you want to introduce periods at the end, please also change all existing translations to avoid follow-up work by others.

Reply:
You can drop it, it's just that most text switch descriptions in Sailfish OS (although not all, so it's not 100% consistent even there!) are full sentences terminated with a period. Sounds like it's not worth the trouble.

Reply:
OK, I'll drop it.